lib/auth.js

```diff
@@ -61,7 +61,10 @@ module.exports = new Model({
   },
 
   _handleNewBearerToken: function(request) {
-    var response = JSON.parse(request.text);
```

Comment:
I think it originally expected the response to be passed directly from the http request - we could parse it first and then call this function, negating the need for this line.